### PR TITLE
Add UEFI Capsule update support

### PR DESCRIPTION
**Pull request by DaniilKl**

### CODE REVIEW

**Comment by DaniilKl** [comment]
This PR depends on merging https://github.com/Dasharo/dts-scripts/pull/26.

**Comment by DaniilKl** [comment]
Some rebasing done

**Comment by DaniilKl** [comment]
Refspec of dts-scripts [will be chenged here](https://github.com/Dasharo/dts-scripts/pull/26#pullrequestreview-2379715381)

**Comment by DaniilKl** [comment]
Should be merged after https://github.com/Dasharo/dts-scripts/pull/26.

File: meta-dts-distro/recipes-dts/dts-scripts/dts-scripts_git.bb [hunk 1]
```diff
@@ -8,7 +8,7 @@ LIC_FILES_CHKSUM = "file://LICENSES/Apache-2.0.txt;md5=c846ebb396f8b174b10ded477
 PV = "0.1+git${SRCPV}"
 
 SRC_URI = "git://github.com/Dasharo/dts-scripts;protocol=https;branch=main"
-SRCREV = "a5f6ef77111cf2dca57888c4a072b6e5d078fa35"
+SRCREV = "875887465755cde563da5dcfe67d5c530664b662"
```
Comment (m-iwanicki) [inline meta-dts-distro/recipes-dts/dts-scripts/dts-scripts_git.bb:11]:
```suggestion
SRCREV = "e757053cae37453cbd7dfaebf0f1a1676829fb25"
```

**Review by m-iwanicki — CHANGES_REQUESTED** [review]

File: meta-dts-distro/recipes-dts/dts-scripts/dts-scripts_git.bb (hunk 1 above)
Comment (DaniilKl) [reply meta-dts-distro/recipes-dts/dts-scripts/dts-scripts_git.bb]: https://github.com/Dasharo/meta-dts/compare/fa25ddb4ed6db4c0e4180425353013f23b3bb287..9de3b6b38940576c1ac4d56d3785e390e9773df5

**Review by m-iwanicki — APPROVED** [review]
LGTM

**Pull request merged**